install.sh

```diff
@@ -0,0 +1,189 @@
+
+#!/usr/bin/env sh
+
+set -eu
+
+if [ -n "${GITHUB_ACTIONS-}" ]; then
+  set -x
+fi
+
+# Check pipefail support in a subshell, ignore if unsupported
+# shellcheck disable=SC3040
+(set -o pipefail 2> /dev/null) && set -o pipefail
+
+help() {
+  cat <<'EOF'
+Install a binary released on GitHub
+
+USAGE:
+    install.sh [options]
+
+FLAGS:
+    -h, --help      Display this message
+    -f, --force     Force overwriting an existing binary
+
+OPTIONS:
+    --repo REPO     Github Repository to install the binary from
+    --bin BIN       Name of the binary to install
+    --tag TAG       Tag (version) of the bin to install, defaults to latest release
+    --to LOCATION   Where to install the binary [default: ~/.cargo/bin]
+    --target TARGET
+EOF
+}
+
+
+say() {
+  echo "install: $*" >&2
+}
+
+err() {
+  if [ -n "${td-}" ]; then
+    rm -rf "$td"
+  fi
+
+  say "error: $*"
+  exit 1
+}
+
+need() {
+  if ! command -v "$1" > /dev/null 2>&1; then
+    err "need $1 (command not found)"
+  fi
+}
+
+
+
+download() {
+  url="$1"
+  output="$2"
+
+  if command -v curl > /dev/null; then
+    curl --proto =https --tlsv1.2 -sSfL "$url" "-o$output"
+  else
+    wget --https-only --secure-protocol=TLSv1_2 --quiet "$url" "-O$output"
+  fi
+}
+
+force=false
+while test $# -gt 0; do
+  case $1 in
+    --help | -h)
+      help
+      exit 0
+      ;;
+    --repo)
+      repo=$2
+      shift
+      ;;
```

Comment:
We should set the `repo`/`bin` to `dora-rs/dora` by default, given that this script is dora-specific. I think we could even hardcode it, as there is probably no reason to use this script to install other projects.

Reply:
So we might have other binaries such as `dora-record` and the likes that we might want to share without having cargo.

On the small computer RK3588 running with OpenHarmony, there is no cargo and it might be really annoying to compile everything everytime.